db/fixtures/miq_shortcuts.yml

```diff
@@ -474,3 +474,13 @@
   :url: /miq_task/index?jobs_tab=tasks
   :rbac_feature_name: tasks
   :startup: true
+- :name: red_hat_cloud_services
```

Comment:
I don't think this belongs in this repo.  We probably need to make seeding pluggable so that it can live in the appropriate plugin.

Reply:
cc @Fryguy did you have a list of things that need to be pluggable?

Reply:
https://github.com/ManageIQ/manageiq/issues/14840 but we closed it a while ago.  We should open a new issue to track the newer stuff we've found.